packages/sortable-behavior-bundle/README.md

```diff
@@ -32,6 +32,31 @@ return [
 ];
 ```
 
+### Use a draggable list instead of up/down buttons
+
+In order to use a draggable list instead of up/down buttons, change the template in the `move` action to `@RunroomSortableBehavior/sort.html.twig`.
```

Comment:
I normally use the default sort, but to use the draggable, I think you have to use another template.

Reply:
done